tests/Doctrine/Migrations/Tests/Metadata/Storage/TableMetadataStorageTest.php

```diff
@@ -98,6 +98,27 @@ public function testComplete() : void
         ], $rows);
     }
 
+    public function testCompleteWithFloatTime() : void
+    {
+        $result = new ExecutionResult(new Version('1230'), Direction::UP, new DateTimeImmutable('2010-01-05 10:30:21'));
+        $result->setTime(31.49);
```

Comment:
Why can we set a float when we need an `int`?

Reply:
Hmm, it should be a float, since it represents microseconds too. Here the issue is how data are saved. Currently it seems that we loose some info (the sub second execution time).
Will update the PR